druid/src/scroll_component.rs

```diff
@@ -124,11 +125,6 @@ impl ScrollbarsState {
 /// [`paint_content`]: struct.ScrollComponent.html#method.paint_content
 #[derive(Debug, Copy, Clone)]
 pub struct ScrollComponent {
```

Comment:
This name now seems even less appropriate than it did before. Maybe in a subsequent PR it could be renamed to ScrollbarOverlay or something. I'm also not entirely sure why its on the top level - even though it is not a widget itself, it is entirely concerned with widget matters as far as I can tell. But can totally see leaving it for this PR to make the diff easier to read

Reply:
I basically agree. But as you know, my dream is to eventually move this logic into a nicely-composable widget. In the meantime, I'm not super inclined to do lots of moving and renaming...